Not working in Windows

Under WSL with Node.js v17, I get the following in both the browser and shell. The file in question exists (it's a copy of a jscad example from https://www.openjscad.xyz/ ?->"Building a Dodecahedron") and can even be accessed from the opened server page.
```
Can't walk dependency graph: Cannot find module 'test_jscad.js' from '/home/david/test/jscad-now.js'
    required by /home/david/test/jscad-now.js
```

Under Windows 10 directly via Powershell in Node v16 I get no errors in the shell, a blank page in the browser, and an exception in the browser javascript console
```
parameterControls.js:19 Uncaught TypeError: Cannot read properties of null (reading 'map')
    at createParamControls (jscad-now.js:39537:41)
    at designParameters (jscad-now.js:39488:24)
    at module.exports (jscad-now.js:39341:45)
    at module.exports (jscad-now.js:39325:3)
    at Object.606../test1.jscad (jscad-now.js:39934:1)
    at o (jscad-now.js:1:265)
    at r (jscad-now.js:1:431)
    at jscad-now.js:1:460
```

Also, fyi, npm is reporting 5 'high severity vulnerabilities'.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "jscad-now",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Instantly view a jscad V2 module",
   "bin": {
     "jscad-now": "bin/cli.js"

--- a/src/client/index.js
+++ b/src/client/index.js
@@ -54,6 +54,8 @@ module.exports = function (gridsize, model, getParameterDefinitions) {
     scope.parameterDefinitions = getParameterDefinitions()
     scope.paramState = getParameterValuesFromParameters(scope.parameterDefinitions)
     scope.entities = getEntities(scope.paramState)
+  } else {
+    scope.entities = getEntities()
   }
 
   // prepare

--- a/src/client/inputs.js
+++ b/src/client/inputs.js
@@ -2,7 +2,7 @@ const createElement = require('./element')
 const designParameters = require('./parameters/web/designParameters')
 
 module.exports = function (scope, callback) {
-    createElement('parameters').appendChild(designParameters(
+    const designParamElements = designParameters(
         {
             design: {
                 parameterValues: scope.paramState,
@@ -12,5 +12,7 @@ module.exports = function (scope, callback) {
         },
         { callback },
         (s) => s)
-    )
+    if (designParamElements) {
+        createElement('parameters').appendChild(designParamElements)
+    }
 }

--- a/src/client/parameters/web/designParameters.js
+++ b/src/client/parameters/web/designParameters.js
@@ -4,6 +4,9 @@ const { createParamControls } = require('./parameterControls')
 
 const designParameters = (state, paramsCallbacktoStream, i18n) => {
   const { parameterValues, parameterDefinitions, parameterDefaults } = state.design
+  if (!parameterDefinitions) {
+    return '';
+  }
   const { controls } = createParamControls(
     Object.assign({}, parameterDefaults, parameterValues), parameterDefinitions, paramsCallbacktoStream.callback
   )

